service.plexskipintro/lib/addon.py

```diff
@@ -0,0 +1,107 @@
+import xbmc, xbmcaddon, xbmcgui
+from threading import Timer
+from plexapi.server import PlexServer
+from lib.definitions import *
+import pprint
+import time
+
+def closeDialog():
+    global Dialog
```

Comment:
What are all these `globals` in this file?

Reply:
these globals are being used to pass around values between defs for various variables such as the times of the intro or the dialog to be displayed/hidden